### ISSUE TITLE
Butterfly Classification Analysis

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#623]
Define You:

- [ ] KWOC 2023 Participant 
- [ ] Contributor

Hey OS participants, good to see you.

This project will help us in analyzing and classifying the various species of butterflies using dataset given

Dataset : https://www.kaggle.com/bertcarremans/butterfly-images

If you like to work on this issue, follow the given guidelines for code as well as for the README. You can try to compare it using different algorithms for better modelling practices.

**Comment by Aryanmartinian** [comment]
Hi I want to contribute to this project under KWOC 23 as Contributor

**Comment by prathimacode-hub** [comment]
Issue assigned to @Aryanmartinian

**Comment by prathimacode-hub** [comment]
Any update? @Aryanmartinian

**Comment by Aryanmartinian** [comment]
Working on it

**Comment by prathimacode-hub** [comment]
Just few hours left for the deadline. Kindly submit your PR at the earliest possible. Thank you. @Aryanmartinian

**Comment by prathimacode-hub** [comment]
Any update? @Aryanmartinian

**Comment by alien-cyber** [comment]
HI i want to contribute to this project under dwoc 23, planning to used Restnet for classification

**Comment by prathimacode-hub** [comment]
Glad to know, as what i had heard from the team the contribution starts from Jan 24th. According to guidelines, before getting it assigned you need to create a proposal. Thanks for showing your interest. If you want to contribute to projects in ML-ProjecKart, submit the proposal before 24th. Only few of them, will be selected for the project. All the very best. @alien-cyber

**Comment by alien-cyber** [comment]
In the guildlines for contributes in dwoc,it is just mentioned comment on the issue. 
Can you tell what proposal and where to submit

**Comment by prathimacode-hub** [comment]
Yes, right. It's the same format for other open source programs. But for this program, you need to write a proposal, introducing yourself, your experience in this field  why are you interested to contribute in this project, what projects you could add, changes you would make, impact you would create. As of now, you just have to create a proposal ready from your end. I heard from team that the project channels will be created on 24th. On 25th, selection of project proposals will be made. Therefore you could start contributing from then on till the duration. This is how dwoc works similar to Google Summer Of Code. If you would like to, I could share the sample format for you to create. @alien-cyber

**Comment by alien-cyber** [comment]
Name : Nanthakumar
Approach to  this project: using resnet to extract image features and connecting it to custom made connected neural network.
I am interested in Ai/DL, so opted for this project,This will be my first ML project

**Comment by prathimacode-hub** [comment]
This should be in form of document that should be shared on project channel. I will send you a template. @alien-cyber

**Comment by alien-cyber** [comment]
Maam,out of curiosity I completed the project 
here is the repo: https://github.com/alien-cyber/Butterfly-classification

**Comment by prathimacode-hub** [comment]
Hey, if you had completed well and good. Glad to hear your spirit. I'll consider you I'm a project for contribution definitely. You can follow the process after the event begins
But make sure you have a backup of your completed project code. Go through project structure and contributing guidelines meanwhile. @alien-cyber

**Comment by alien-cyber** [comment]
Thank you,This project becomes easier because you already gave me the dataset
https://github.com/alien-cyber/Butterfly-classification

**Comment by adi271001** [comment]
please assign this to me under dwoc

**Comment by dipayan22** [comment]
Name : Dipayan Majumder
github : dipayan22
email : majumderd007@gmail.com

i am interest to contribute in this project . Can you assign me this project under DWOC 24.

**Comment by prathimacode-hub** [comment]
Issue assigned. You can proceed. @dipayan22

**Comment by pavitraag** [comment]
Can you pease assign this to me

**Comment by prathimacode-hub** [comment]
Issue assigned to @pavitraag

**Comment by prathimacode-hub** [comment]
Update please? @pavitraag

**Comment by prathimacode-hub** [comment]
Since there is no update of PR from your end, I'm unassigning you. As well, CodePeak 2025 program is wrapped up. All the best for your future endeavours.. @pavitraag
